aten/src/ATen/cuda/Atomic.cuh

```diff
@@ -223,14 +223,13 @@ static inline  __device__ at::Half gpuAtomicAdd(at::Half *address, at::Half val)
 }
 
 static inline __device__ at::BFloat16 gpuAtomicAdd(at::BFloat16 *address, at::BFloat16 val) {
-#if defined(USE_ROCM) || ((defined(__CUDA_ARCH__) && (__CUDA_ARCH__ < 800)))
-return AtomicFPOp<at::BFloat16>()(address, val,
-                                  [](at::BFloat16 bsum, at::BFloat16 val) {
-                                    return bsum + val;
-                                  });
+#if defined(USE_ROCM) || ((defined(CUDA_VERSION) && CUDA_VERSION < 11000) || (defined(__CUDA_ARCH__) && (__CUDA_ARCH__ < 800)))
```

Comment:
Could we use the current code as is since I believe there are no real changes here?